src/pages/Manager.vue

```diff
@@ -938,7 +929,7 @@ import GameRunningModal from '../components/modals/GameRunningModal.vue';
                     this.setAllModsEnabled(false);
                     break;
                 case "UpdateAllMods":
-                    this.showUpdateAllModal = true;
+                    this.$store.commit("openDownloadModModal");
```

Comment:
I'm not a big fan of implicitly assuming the context is to update all mods if no specific mod is set, mostly due to it not being very obvious from the code if you don't check. I'm not sure what would I expect a mod download modal to do if it doesn't get a mod assigned to it, but updating all mods is not it at least.

I understand that logic from the state-machine standpoint, as it ensures the modal can't end up in an invalid state, but I'd suggest exploring some ways to make it more obvious to the developer (naming? idk).

Perhaps using two separate transactions at least, one for opening the modal in update view and one for a specific mod?

Also shouldn't the commit use a namespaced transaction name, i.e. `"modalsModule/openDownloadModModal"` or whatever the namespace was?

Reply:
Changed to have `openDownloadModModal` for a single mod and `openUpdateAllModsModal` for updating all.

> Also shouldn't the commit use a namespaced transaction name, i.e. "modalsModule/openDownloadModModal" or whatever the namespace was?

Vuex supports choosing whether to namespace submodules on a module basis. In this case I chose not to, since I figured modal management is something that happens globally and the calls aren't limited to a single package or feature. So if the question was "is the lack of namespace a bug", the answer is no. If it was more "I'd like this to be namespaced", it can be done if we think there should be a convention to always use namespaces.